hw14/src/test/java/base/Hw14.java

```diff
@@ -0,0 +1,77 @@
+package base;
+
+import org.openqa.selenium.Beta;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class Hw14 extends BaseTest {
+    List<String> files = Files.readAllLines(Paths.get(this.getClass().getClassLoader().getResource("LogDate.properties").toURI()));
+
+    public Hw14() throws IOException, URISyntaxException {
+    }
+
+    @Test
+
+    public void validLog() throws InterruptedException {
+
+        driver.findElement(By.cssSelector("a[onclick='return i_showFloat(1);']")).click();
+        driver.findElement(By.cssSelector("fieldset input[name='login']")).sendKeys(files.get(0));
+        driver.findElement(By.cssSelector("fieldset input[name='pass']")).sendKeys(files.get(1));
+        driver.findElement(By.cssSelector("input:nth-child(7)")).click();
+        WebElement searchResult = driver.findElement(By.cssSelector(".right a[title='выйти']")); //Получаем результат поиска

```

Comment:
Please fix comments and variable names to match with meanings (according to comments in previous pull request)